From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Doe <john.doe@somewhere.on.planet>
Date: Tue, 30 Jan 2024 19:44:13 +0100
Subject: Patching kernel mvebu files
 drivers/net/dsa/mv88e6xxx/mv88e6xxx_debugfs.cDisable dsa-port slave print

Signed-off-by: John Doe <john.doe@somewhere.on.planet>
---
 drivers/net/dsa/mv88e6xxx/mv88e6xxx_debugfs.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/dsa/mv88e6xxx/mv88e6xxx_debugfs.c b/drivers/net/dsa/mv88e6xxx/mv88e6xxx_debugfs.c
index 4005a4760884..a5cba5a2965a 100644
--- a/drivers/net/dsa/mv88e6xxx/mv88e6xxx_debugfs.c
+++ b/drivers/net/dsa/mv88e6xxx/mv88e6xxx_debugfs.c
@@ -218,14 +218,14 @@ static int mv88e6xxx_name_show(struct seq_file *s, void *p)
 		i = dp->index;
 		if (!ds->cd->port_names[i])
 			continue;
 
 		seq_printf(s, "%4d   %s", i, ds->cd->port_names[i]);
-
+#if 0
 		if (dp->slave)
 			seq_printf(s, " (%s)", netdev_name(dp->slave));
-
+#endif
 		seq_puts(s, "\n");
 	}
 
 	return 0;
 }
-- 
Created with Armbian build tools https://github.com/armbian/build

